code/frontend/src/app/ui/drawer/drawer.component.scss

```
@@ -0,0 +1,96 @@
+  inset: 0;
+  background: rgba(0, 0, 0, 0.6);
+  z-index: var(--z-modal);
+  display: flex;
+  justify-content: flex-end;
+  animation: fade-in var(--duration-fast) var(--ease-default),
+             backdrop-blur-in var(--duration-normal) var(--ease-default) both;
+}
+
+.drawer {
+  @include glass('elevated');
+  position: relative;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)